packages/core/src/components/popover/_common.scss

```diff
@@ -1,7 +1,7 @@
 // Copyright 2015 Palantir Technologies, Inc. All rights reserved.
 // Licensed under the terms of the LICENSE file distributed with this project.
 
-@import "~bourbon/app/assets/stylesheets/bourbon";
+
```

Comment:
Probably don't need two newlines in a row here.

Reply:
ah snap one slipped through!